### DISCUSSION TITLE
라이브 핸즈온 워크샵 일정 및 발표자료

### DISCUSSION THREAD

**Discussion by justinyoo** [devrel-kr/HackaLearn#32] [0 공지사항]
HackaLearn 이벤트에 참여하시는 분들 중, 아직 [애저 정적 웹 앱](https://aka.ms/hackalearn/aswa/intro)이랑 [깃헙 액션](https://aka.ms/hackalearn/gha/intro)이 낯선 분들 많이 계시죠? 그래서 준비했습니다.

| 제목 | 일시 | 스트리밍 | 슬라이드 | 샘플코드 |
| :--: | :--: | :--: | :--: | :--: |
| [GCE][gce]와 함께하는 깃헙 액션 만들기 | 2021년 8월 3일 (화) 8pm | [바로 가기][live gce] | [다운로드][slide gce] | [링크][code gce] |
| [MLSA][mlsa]와 함께하는 애저 정적 웹 앱 만들기 | 2021년 8월 5일 (목) 8pm | [바로 가기][live mlsa] | [다운로드][slide mlsa] | [링크][code mlsa] |
| [클라우드 아드보캇][ca]과 함께하는 블로그 사이트 만들기 | 2021년 8월 10일 (화) 8pm | [바로 가기][live ca] | [다운로드][slide ca] | [링크][code ca] |

많은 관심과 참여 부탁 드립니다!

[mlsa]: https://aka.ms/hackalearn/mlsa
[gce]: https://aka.ms/hackalearn/gce
[ca]: https://aka.ms/hackalearn/ca

[live mlsa]: https://youtu.be/Hxkv6AjAisY
[live gce]: https://youtu.be/e_elLW6uNSc
[live ca]: https://youtu.be/x3j3mDblqMY

[slide mlsa]: https://aka.ms/hackalearn/aswa/hol
[slide gce]: https://aka.ms/hackalearn/gha/hol
[slide ca]: https://aka.ms/hackalearn/hol/slides

[code mlsa]: https://aka.ms/hackalearn/aswa/code
[code gce]: https://aka.ms/hackalearn/gha/code
[code ca]: https://aka.ms/hackalearn/hol/code

**Comment by loreenkim** [comment]
해커톤에서 새로운 걸 학습까지 할 수 있다니 훌륭하네요~ :)
